demos/browser/server.js

```diff
@@ -62,7 +62,7 @@ function serve(host = '127.0.0.1:8080') {
       } else if (process.env.DEBUG) {
         // For internal debugging - ignore this
         const debug = require('./debug.js');
```

Comment:
Could you remind me where we get this file?

Reply:
I'll send you a DM